docs/source/model_design/vivarium_model_components/risk_factors/risk_correlation/index.rst

```diff
@@ -1,1213 +1,152 @@
-        </tr>
-        <tr>
-          <th>9997</th>
-          <td>129.865889</td>
-          <td>5.465636</td>
-        </tr>
-        <tr>
-          <th>9998</th>
-          <td>105.705538</td>
-          <td>3.818431</td>
-        </tr>
-        <tr>
-          <th>9999</th>
-          <td>143.099773</td>
-          <td>7.605454</td>
-        </tr>
-      </tbody>
-    </table>
-    <p>10000 rows × 2 columns</p>
-    </div>
-
-
-
-.. code:: python
-
-    sns.jointplot(df['SBP'], df['LDL-C'])
-
-
-
-
-.. parsed-literal::
-
-    <seaborn.axisgrid.JointGrid at 0x21892019648>
-
-
-
-
-.. image:: 2020_02_11a_correlation_and_paf_files/2020_02_11a_correlation_and_paf_5_1.png
-
-
-What we want from the microdata is simply the Spearman R value, which captures the correlation of SBP and LDL-C as continuous variables
-=======================================================================================================================================
-
-.. code:: python
-
-    rho, p_val = scipy.stats.spearmanr(df['SBP'], df['LDL-C'])
-    rho
-
-
-
-
-.. parsed-literal::
-
-    0.6942625526106255
-
-
-
-What will we use that for?
-==========================
-
-We will generate a multivariate normal with that rho, and then use that
-to generate a distribution that is marginally uniform, but with that
-rho, and then use that to generate a distribution that in its margins
-matches the SPB and LDL-C from GBD, but also has Spearman correlation
-rho:
-
-.. code:: python
-
-    probit_propensity = np.random.multivariate_normal([0,0], [[1, rho],
-                                                             [rho, 1]], size=N)
-    sns.jointplot(probit_propensity[:,0], probit_propensity[:,1])
-    rho, p_val = scipy.stats.spearmanr(probit_propensity[:,0], probit_propensity[:,1])
-    rho
-
-
-
-
-.. parsed-literal::
+Instead of sampling from a univariate distribution for each risk exposure at initialization,
+Vivarium can sample multiple exposures simultaneously from a multivariate distribution.
+For example, given a correlation coefficient between two continuous exposures, it can sample from a bivariate normal distribution
+instead of two independent univariate normal distributions.
 
-    0.6797538355495384
+We generally cannot get multivariate distributions from the GBD;
+instead, we seek out auxiliary data sources.
+These may report summary statistics such as a correlation coefficient, or
+have full microdata we can use to summarize the joint distribution in a more detailed way.
+However, no matter how we characterize the relationship, we prefer to continue to validate to the GBD marginal distributions,
+since we trust them more than any single data source.
 
+This plot shows data from a joint distribution reflecting the variance and covariance between height and weight for infants at age one month using
+data from a cohort study.
+The top panel shows the values for the height and weight z-scores and the bottom panel shows the percentiles
```

Comment:
Technically, the top panel shows histograms of the percentiles/quantiles (i.e. the actual values of WAZ and HAZ), just not labeled as such, and the bottom panel shows histograms of the **percentile ranks**/**quantile ranks**. See these Wikipedia pages:
- https://en.wikipedia.org/wiki/Percentile
- https://en.wikipedia.org/wiki/Quantile
- https://en.wikipedia.org/wiki/Percentile_rank

Reply:
I do not understand what you mean when you say the top panel uses quantiles. It shows Z scores, which are still normally distributed.

I think that "percentile rank" is very commonly referred to as "percentile" (e.g. by the person who made this graph) but the difference makes sense to me and I will add it to the text here, but not mess with the graph.

Reply:
> I do not understand what you mean when you say the top panel uses quantiles. It shows Z scores, which are still normally distributed

Yeah, that's kind of confusing, I was just trying to make a point. What I mean is that if your height (H) is 5'10", and that's the 70th percentile (say) of the distribution of H, then the 70th percentile of H is 5'10", and the percentile rank of 5'10" is 70%. So if you draw a histogram of H, then every value on the x-axis is some percentile of H, because the percentiles of H are simply *values* of H, just labeled with their percentile rank (e.g., the value h=5'10" can be labeled as the 70th percentile). So you're drawing a histogram with "percentiles" on the x-axis, except that we don't call them percentiles in the context of drawing a histogram. (I don't know if that really makes sense, but I don't see any flaws in the logic...)

> I think that "percentile rank" is very commonly referred to as "percentile" (e.g. by the person who made this graph) but the difference makes sense to me and I will add it to the text here, but not mess with the graph.

Yeah, when we started writing the docs, some confusion arose because we were referring to both the propensities and the corresponding exposure values as percentiles, so we consciously tried to make the distinction between quantiles and quantile ranks in order to be able to communicate more clearly with each other.